pkgs/development/libraries/qt-6/modules/qtshadertools.nix

```diff
@@ -1,8 +1,18 @@
 { qtModule
 , qtbase
+, stdenv
+, lib
+, pkgsBuildBuild
 }:
 
+let
+  isCrossBuild = stdenv.hostPlatform != stdenv.buildPlatform;
```

Comment:
Should be `!stdenv.buildPlatform.canExecute stdenv.hostPlatform`, also don't have to be in a let block since it's only used once.

Reply:
Are you saying do something like 
```nix
qtModule rec {
  isCrossBuild = ...
```

Reply:
I think I made the change you wanted